test/GovernanceStaker.t.sol

```diff
@@ -2764,6 +2764,58 @@ contract NotifyRewardAmount is GovernanceStakerRewardsTest {
     assertLe(govStaker.scaledRewardRate(), _expectedRewardRate);
   }
 
+  function test_shareManipulationIfRewardsArentCeckpointed() public {
```

Comment:
```suggestion
  function test_shareManipulationIfRewardsArentCheckpointed() public {
```

Reply:
should we rename this test to describe that there's no longer share manipulation? also, are there unit tests to write here (e.g. alterDelegatee checkpoints global reward, alterDelegatee checkpoints reward)? defer to you on that